node/src/actors/chain_manager/mod.rs

```diff
@@ -374,10 +374,12 @@ impl ChainManager {
                             self.broadcast_item(InventoryItem::Block(block));
                             log::debug!("Send Candidate");
                         }
-                        Err(e) => log::warn!("{}", e),
+                        Err(e) => log::warn!("Validate candidate {}: {}", hash_block, e),
                     },
-                    Err(e) => log::warn!("{}", e),
+                    Err(e) => log::warn!("Validate candidate {}: {}", hash_block, e),
```

Comment:
What's the difference between this error and the previous one?

Reply:
This inner one validates the block epoch and the block hash_prev_block, and the second one validates the VRF proof. They are two steps of the same validation, so I will try to merge the two logs into one.